app/components/post_list/post_list.js

```diff
@@ -35,6 +35,10 @@ export default class PostList extends Component {
         lastViewedAt: PropTypes.number
     };
 
+    static defaultProps = {
```

Comment:
also in the renderChannelIntro now it will try and render it always as channel will always exist, I think you need to remove this or come up with another way to test for channel in the renderIntro